appcues/src/main/java/com/appcues/data/mapper/experience/ExperienceMapper.kt

```diff
@@ -89,10 +89,10 @@ internal class ExperienceMapper(
         return filterIsInstance<PresentingTrait>().firstOrNull() ?: DefaultPresentingTrait(null, scope, context)
     }
 
-    private fun Map<String, ExperimentResponse>.getExperiment(experimentId: String?) =
+    private fun List<ExperimentResponse>.getExperiment(experimentId: String?) =
         experimentId?.let { id ->
-            this[id]?.let { experimentResponse ->
-                Experiment(id, experimentResponse.group)
+            this.firstOrNull { it.experimentId == id }?.let { experimentResponse ->
+                Experiment(experimentResponse.experimentId, experimentResponse.group)
```

Comment:
I assume there can only be one item in the array with a given `experimentId`, but confirming this.  The previous mapping was more clear in this sense.